doc/api/v8.md

```diff
@@ -271,6 +271,89 @@ buffers and external strings.
 }
 ```
 
+## `v8.getCppHeapStatistics(detailLevel)`
+
+Retrieves [CppHeap][] regarding memory consumption and
+utilization statistics using the V8
+[`CollectStatistics()`][] function which may change from one V8 version to the
+next.
+
+* `detailLevel` (`'brief'`|`'detailed`): Specifies the level of detail in the
+  returned statistics.
+  * `brief`:  Brief statistics contain only the top-level allocated and used
+    memory statistics for the entire heap.
+  * `detailed`: Detailed statistics also contain a break down per space and
+    page, as well as freelist statistics and object type histograms.
```

Comment:
```suggestion
* `detailLevel` {string|undefined}:  **Default:** `'detailed'`. Specifies the level of detail in the returned statistics.
  Accepted values are:
  * `'brief'`:  Brief statistics contain only the top-level allocated and used
    memory statistics for the entire heap.
  * `'detailed'`: Detailed statistics also contain a break down per space and
    page, as well as freelist statistics and object type histograms.
```

I am not exactly sure what's the best way to document these enums, but we do have some precedents of this style in the docs.

Reply:
There are similar examples in v8.md, for example here: https://github.com/nodejs/node/blob/9ce1fffcdc5a6045e19c6730aa6ae3e271287b23/doc/api/v8.md?plain=1#L286